Makefile

```diff
@@ -23,9 +23,9 @@ help:
 .PHONY: dev
 dev:
 ifneq ($(LUAROCKS_VER),'luarocks 3')
-	luarocks install rockspec/orange-0.8-0.rockspec --tree=deps --only-deps --local
+	luarocks install rockspec/orange-dev-1.0-0.rockspec --tree=deps --only-deps --local
```

Comment:
orange-dev-1.0 -> orange-master-1.0

how about this name? ^_^

Reply:
very good, this is more reasonable.